package.json

```diff
@@ -21,13 +23,21 @@
   "scripts": {
     "compile": "tsc --noEmit",
     "format": "prettier --ignore-path .prettierignore --write",
+    "format:md": "prettier --ignore-path .prettierignore --write \"./**/*.(md)\"",
     "format:all": "prettier --ignore-path .prettierignore --write \"./**/*.(ts|md)\"",
     "format:verify": "prettier \"./**/*.(ts|md)\" --list-different",
     "lint:js": "eslint \"./**.ts\" --max-warnings 0 ",
     "lint:md": "markdownlint \"./content/**/*.md\"",
     "lint": "yarn lint:md && yarn lint:js",
-    "test": "jest"
+    "test": "jest",
+    "prepare": "husky install"
   },
   "license": "UNLICENSED",
-  "version": "1.0.0"
+  "version": "1.0.0",
+  "lint-staged": {
+    "*.md": [
+      "npm run format",
+      "markdownlint"
+    ]
+  }
```

Comment:
Nittiest of nits: up for moving the `package.json` keys towards being alphabetically sorted by moving this up? 😄

Reply:
@JoshuaKGoldberg how do things look now? e7a1ee0320b4a5103c5793c6e2484c918557bf2f

Reply:
🎉